src/faldbt/project.py

```diff
@@ -163,9 +163,9 @@ def get_sources(self) -> List[ParsedSourceDefinition]:
         return list(self.nativeManifest.sources.values())
 
 
-@dataclass(init=False)
 class DbtRunResult:
-    nativeRunResult: Optional[RunResultsArtifact]
+    """Holds dbt run result information."""
+
     results: Sequence[RunResultOutput]
```

Comment:
Should we remove this one too?

Reply:
It's back to being a dataclass, because of the auto-generated __repr__ function.